bazel/repository_locations.bzl

```diff
@@ -802,9 +802,9 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_desc = "Open-source distributed event streaming platform",
         project_url = "https://kafka.apache.org",
         version = "2.4.1",
-        sha256 = "2177cbd14118999e1d76fec628ca78ace7e6f841219dbc6035027c796bbe1a2a",
-        strip_prefix = "kafka_2.12-{version}",
-        urls = ["https://mirrors.gigenet.com/apache/kafka/{version}/kafka_2.12-{version}.tgz"],
+        sha256 = "d1064e87c2f1aee64f659ca9fd9d491d6d35b37e3bdb1fbd588d6100111e5e49",
+        strip_prefix = "kafka-{version}-src",
+        urls = ["https://archive.apache.org/dist/kafka/{version}/kafka-{version}-src.tgz"],
```

Comment:
isn't that the change from compiled binary to zip with sources?
could you please try to run
`bazel test //test/extensions/filters/network/kafka/broker/integration_test:kafka_broker_integration_test`

Reply:
Makes sense, I think you are right. Changing it to the binaries